File: programers_Lv1/이상한 문자 만들기.js
// https://school.programmers.co.kr/learn/courses/30/lessons/12930?language=javascript
function solution(s) {
  const sArr = s.split(" ");
  const answer = sArr
    .map((el) =>
      [...el]
        .map((v, i) => (i % 2 === 0 ? v.toUpperCase() : v.toLowerCase()))
        .join("")
    )
    .join(" ");

  return answer;
}

console.log(solution("try hello world"));
console.log(solution("a  b  c"));

File: programers_Lv1/이상한 문자 만들기.py
# https://school.programmers.co.kr/learn/courses/30/lessons/12930#qna
def solution(s):
    s_list = s.split(' ')
    answer = ''
    for word in s_list:
        for i, v in enumerate(word):
            if i % 2 == 0:
                answer += v.upper()
            else:
                answer += v.lower()
        answer += ' '

    return answer[:-1]

print(solution("try hello world"))
print(solution("a  b  c"))
